### PR TITLE
Add fakerealtime example.

### PR DESCRIPTION
**Pull request by bdarnell**
Moved from cockroachdb/cockroach#2627.

### CODE REVIEW

**Comment by mberhault** [comment]
LGTM

File: fakerealtime/main.go [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+	message := start.String()
+
+	// TODO(bdarnell): retry only on certain errors.
+	for {
+		txn, err := w.db.Begin()
+		if err != nil {
+			continue
+		}
+
+		// TODO(bdarnell): make this a subquery when subqueries are supported on insert.
```
Comment (petermattis) [inline fakerealtime/main.go]: FYI, subqueries are supported on insert now.

File: fakerealtime/main.go [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+		}
+		newUpdateID := maxUpdateID.Int64 + 1
+
+		if _, err := txn.Exec(`insert into fakerealtime.messages (channel, msg_id, message) values ($1, $2, $3)`,
+			channel, newMsgID, message); err != nil {
+			_ = txn.Rollback()
+			continue
+		}
+
+		if _, err := txn.Exec(`insert into fakerealtime.updates (update_id, channel, msg_id) values ($1, $2, $3)`,
```
Comment (tschottdorf) [inline fakerealtime/main.go]: can't you exec this statement together with the previous?

Comment (bdarnell) [inline fakerealtime/main.go]: I could, but that's cockroach-specific and I wanted to at least have the option to test this against other databases.

**Pull request merged**